src/main/steps/common/components/radios.ts

```diff
@@ -0,0 +1,73 @@
+import { TranslationFn } from '../../../app/controller/GetController';
+import { FormContent, FormInput } from '../../../app/form/Form';
+import { isFieldFilledIn } from '../../../app/form/validation';
+import { CommonContent } from '../common.content';
+
+import { defaultButtons } from './default-buttons';
+
+export class Radios {
+  enContent: Record<string, unknown>;
+  cyContent: Record<string, unknown>;
+  fieldName: string;
+  values: Record<string, string>;
+  form: FormContent;
+
+  constructor(
+    enContent: Record<string, unknown>,
+    cyContent: Record<string, unknown>,
+    fieldName: string,
+    values: Record<string, string>
+  ) {
+    this.enContent = enContent;
+    this.cyContent = cyContent;
+    this.fieldName = fieldName;
+    this.values = values;
+    this.form = this.generateForm();
+  }
+
+  languages = {
+    en: (): Record<string, unknown> => ({
+      ...this.enContent,
+      yes: 'Yes',
+      no: 'No',
+      unsure: 'Not sure',
```

Comment:
these texts are already declared in `common.content.ts` could you think of a way to reuse the same